### PR TITLE
Work Plan Report Dashboard

### PR DESCRIPTION
**Pull request by keeysnc**
### Description
Set up for work plan report dashboard ui. 

### Related ticket(s)
https://qmacbis.atlassian.net/jira/software/c/projects/MDCT/boards/232?modal=detail&selectedIssue=MDCT-2757
MDCT-2757

---
### How to test
1. Login 
2. Go to http://localhost:3000/wp (there's a bug on landing page where work plan goes to "/mp/get-started"). I'll be making a new but ticket for this. 

---
### Author checklist

- [ ] I have performed a self-review of my code
- [ ] I have added [thorough](https://bit.ly/3zPrxuZ) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

File: services/ui-src/src/components/app/AppRoutes.tsx [hunk 1]
```diff
@@ -20,6 +21,8 @@ export const AppRoutes = () => {
         <Route path="/standard" element={<ReportPageWrapper />} />
         <Route path="/reviewSubmit" element={<ReviewSubmitPage />} />
         <Route path="*" element={<NotFoundPage />} />
+        {/* MFP ROUTES */}
+        <Route path="/wp" element={<DashboardPage reportType="WP" />} />
```
Comment (karla-vm) [inline services/ui-src/src/components/app/AppRoutes.tsx:25]: Could we add a new report type in the `types` and use it as a constant?

Comment (karla-vm) [reply services/ui-src/src/components/app/AppRoutes.tsx:25]: Actually i see you have this added already!

Comment (karla-vm) [inline services/ui-src/src/components/app/AppRoutes.tsx:25]:
```suggestion
        <Route path="/wp" element={<DashboardPage reportType={ReportType.WP} />} />
```

and import the ReportType types?

**Review by karla-vm — DISMISSED** [review]
Looks great!! 💯

File: services/ui-src/src/components/forms/Form.tsx [hunk 2]
```diff
@@ -17,14 +17,18 @@ import {
   hydrateFormFields,
   mapValidationTypesToSchema,
   sortFormErrors,
+  useUser,
```
Comment (BearHanded) [inline services/ui-src/src/components/forms/Form.tsx:20]: @karla-vm should this be using the new useUserState?

Comment (karla-vm) [reply services/ui-src/src/components/forms/Form.tsx:20]: ah, yes, `useUserStore`. @keeysnc you might need to merge changes from `main` to pull in the state management stuff.

Comment (keeysnc) [reply services/ui-src/src/components/forms/Form.tsx:20]: Ah okay, thanks.

**Review by karla-vm — APPROVED** [review]

**Review by ajaitasaini — APPROVED** [review]

**Pull request merged**